src/UI/Factory.php

```diff
@@ -40,11 +44,16 @@
 	 *     1: Icons MUST be used to represent objects or context.
 	 *     2: Icons MUST be used in combination with a title or label.
 	 *     3: An unique Icon MUST always refer to the same thing.
+	 *     4: Preconditioned objects whose precondition is not met yet
+	 *        feature Disabled Icons
+	 *     5: Objects to which a user has no read access features Disabled Icons.
```

Comment:
These imo should go to the `context`-field (have a look [here](https://github.com/ILIAS-eLearning/ILIAS/blob/release_5-3/src/UI/docu/rules.md#interfaces-to-factories)).

Reply:
@klees Could you please explain why the context field is never shown in the KS documentation, although it exists in the code? I attached two screenshots to show you what I mean. As you can see, also the other fields are missing, e.g. background in this example. Only the fields description and rules are shown in the documentation.

![ks_context](https://user-images.githubusercontent.com/35489053/40667017-62635b1c-6361-11e8-89d1-83fda6fb90a3.png)
![ks_context_missing](https://user-images.githubusercontent.com/35489053/40667022-69cc02f0-6361-11e8-85e9-8487699d41ee.png)

Reply:
@tfamula-ilias you are right, they are missing. Please create a mantis ticket. This is a bug, which @klees has nothing to do with (system styles are manged by me). You can directly assign this to me. Thx.